README.md

```diff
@@ -1,5 +1,9 @@
 [![build status](https://github.com/maevsi/maevsi/workflows/CI/badge.svg)](https://github.com/maevsi/maevsi/actions?query=workflow%3ACI "build status")
-[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/maevsi/maevsi.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/maevsi/maevsi/context:javascript)
+![Snyk Vulnerabilities for GitHub Repo](https://img.shields.io/snyk/vulnerabilities/github/maevsi/maevsi)
+[![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/maevsi/maevsi)](https://github.com/maevsi/maevsi/issues)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/maevsi/maevsi)
+![GitHub top language](https://img.shields.io/github/languages/top/maevsi/maevsi)
```

Comment:
I'd prefer to keep it clean and only show the build status then. Thank you for showing those banner possibilities though, much appreciated! :green_heart:

Reply:
Sure. The top language tag doubles on githubs repo insights anyway. Sadly, there are no fitting replacements for the code quality rating.